tests/test_prod_info.py

```diff
@@ -26,7 +26,7 @@
 RESTOCK_LEVEL = 'restock_level'
 RESTOCK_AMT = 'restock_amt'
 
-DATABASE_URI = os.getenv('DATABASE_URI', 'sqlite:///db/test.db')
+DATABASE_URI = os.getenv('DATABASE_URI', )
```

Comment:
Maybe remove this line if it is not used at all?

Reply:
addressed